functions/Export-DbaUser.ps1

```diff
@@ -210,7 +210,9 @@ function Export-DbaUser {
                 Write-ProgressHelper -TotalSteps $users.Count -Activity "Exporting from $($db.Name)" -StepNumber ($stepCounter++) -Message "Generating script for user $dbuser"
 
                 #setting database
-                $outsql += "USE [" + $db.Name + "]"
+                if (($PSBoundParameters.ContainsKey('ScriptingOptionsObject') -and $ScriptingOptionsObject.IncludeDatabaseContext) -or -not $PSBoundParameters.ContainsKey('ScriptingOptionsObject')) {
```

Comment:
```suggestion
                if ( ((Test-Bound ScriptingOptionsObject) -and $ScriptingOptionsObject.IncludeDatabaseContext ) -or (Test-Bound ScriptingOptionsObject -Not) ) {
```

We use the internal function across most of our commands so would like to keep this standard.

Reply:
Thanks - I thought there was one but couldn't remember the name. I've updated it, but trying to fix my Pester tests first.